packages/keystone/errors.js

```diff
@@ -133,22 +134,60 @@ class GQLError extends ApolloError {
         if (typeof fields.messageInterpolation !== 'undefined' && (isEmpty(fields.messageInterpolation) || !isObject(fields.messageInterpolation))) throw new Error('GQLError: wrong messageInterpolation argument type! fields.messageInterpolation should be an object')
         if (typeof fields.correctExample !== 'undefined' && typeof fields.correctExample !== 'string') throw new Error('GQLError: wrong correctExample argument type! fields.correctExample should be a string')
         if (typeof fields.context !== 'undefined') throw new Error('GQLError: wrong context argument position! You should pass it as the second argument')
+        if (typeof parentErrors !== 'undefined' && !isArray(parentErrors) && !isError(parentErrors)) throw new Error('GQLError: wrong parent errors argument type')
```

Comment:
How can error have multiple parents?

Reply:
Or by parentErrors you mean entire stack?

Reply:
During GQL call we can receive multiple errors for one query.

Reply:
But they should have single parent, shouldn't they?

Like:
`unable_to_create` -> `unable to connect` -> [`validationError1`, `validationError2`]

Reply:
What's the case of multiple parents?

Reply:
Complex example: 

 GQL Q1 -> [GQL Q2, GQL Q3] -> some errors

```
 - Q1 `GQLError(..[parent2, parent3])
    - Q2 `parent2 = GQLError(..[validationError1, validationError2])`
      - `validationError1`
      - `validationError2`
    - Q3 `parent3 = GQLError(..[validationError3, validationError4])`
      - `validationError3`
      - `validationError4`
```

Reply:
Any sub GQL request can return more than one error by design!

Reply:
You can also check `packages/codegen/generate.server.utils.js` file

Reply:
So, in your example there's only 1 way to reach Q1:
- Q1 has no parents
- Q2 / Q3 has Q1 as parent
- validationError1 / validationError2 has Q2 as parent
- validationError3 / validationError4 has Q3 as parent

So where are the multiple parents occurs?